explore/src/main/scala/explore/targeteditor/SiderealTargetEditor.scala

```diff
@@ -73,18 +77,51 @@ object SiderealTargetEditor {
 
   implicit val propsReuse: Reusability[Props] = Reusability.derive
 
+  def readonlyView[A](view: View[A]): View[A] = {
```

Comment:
Should this go to crystal?

Reply:
It's possible. I thought we could run it by Raul. There may be a better way to do it.